openshift/core.app.yaml

```diff
@@ -0,0 +1,166 @@
+kind: Template
+apiVersion: v1
+metadata:
+  name: core
+objects:
+- kind: DeploymentConfig
+  apiVersion: v1
+  metadata:
+    labels:
+      service: core
+    name: core
+  spec:
+    replicas: 4
+    selector:
+      service: core
+    strategy:
+      resources: {}
+      rollingParams:
+        intervalSeconds: 1
+        maxSurge: 25%
+        maxUnavailable: 25%
+        timeoutSeconds: 600
+        updatePeriodSeconds: 1
+      type: Rolling
+    template:
+      metadata:
+        labels:
+          service: core
+      spec:
+        containers:
+        - command:
+          - /usr/local/alm/bin/alm
```

Comment:
@aslakknutsen I believe we need to remove the `command` due to the changes in the docker image https://github.com/almighty/almighty-core/blob/master/alm%2Bpmcd.sh#L41